src/main/java/com/sforce/cd/apexUnit/client/fileReader/ApexManifestFileReader.java

```diff
@@ -57,6 +57,11 @@
 public class ApexManifestFileReader {
 	private static Logger LOG = LoggerFactory.getLogger(ApexManifestFileReader.class);
 	public static ArrayList<String> nonExistantApexClassEntries = new ArrayList<String>();
+	private boolean includeTriggers;
+
+	public ApexManifestFileReader(boolean includeTriggers) {
```

Comment:
@adarsh-ramakrishna-sfdc  Regarding the purpose of boolean parameter, I guess its intuitive by the name of the boolean variable that it is used to includeTriggers or not. So since it is a boolean parameter, its only add triggers or not. 

Calling *new ApexManifestFileReader()* results in syntax error as the default constructor cannot be called, you need to add a boolean.